tools/@aws-cdk/enum-updater/README.md

```diff
@@ -0,0 +1,104 @@
+# CDK Enum Updater
+
+This tool updates CDK enums with missing enum values on a weekly basis.
+
+To run the tool locally, run the following commands to install dependencies and build:
+
+```
+yarn install
+yarn build
+```
+
+To run the logic for updating the static mapping file, `lib/static-enum-mapping.json`, run the following command:
+```
+./bin/update-static-enum-mapping
+```
+
+The logic for this step is in `lib/static-enum-mapping-updater.ts`
+
+
+To run the logic for identifying missing enum values and creating the code changes, run the following command:
+```
+./bin/update-missing-enums
+```
+
+the logic for this step is in `lib/missing-enum-updater.ts`
+
+# Exclusions
+Exclusions should be added to the `/lib/exclude-values.json` file. There are 2 cases where an exclusion may be needed:
+
+1. If an invalid static mapping is being created in `/lib/static-enum-mapping.json`
```

Comment:
nit: for consistency with the above lines, otherwise changing those above lines to `/lib/...` instead of `lib/...`. Might also be worthwhile to provide links to the files as well? But I feel the name of the file alone is probably descriptive enough.
```suggestion
Exclusions should be added to the `lib/exclude-values.json` file. There are 2 cases where an exclusion may be needed:

1. If an invalid static mapping is being created in `lib/static-enum-mapping.json`
```